content/40.inequality.md

```diff
@@ -42,27 +46,29 @@ ACE2 expression in the kidneys of male mice was observed to be twice as high as
 [@doi:10.1111/dom.14057] and [@doi:10.1152/ajpendo.00124.2020] discuss possible ways in which COVID-19 and diabetes may interact.
 Obesity also appears to be associated with higher risk of severe outcomes from SARS-CoV-2 [@doi:10.1016/j.metabol.2020.154262; @doi:10.1101/2020.04.23.20076042].
 Obesity is considered an underlying risk factor for other health problems, and the mechanism for its contributions to COVID-19 hospitalization or mortality is not yet clear [@doi:10.1016/j.medj.2020.06.005].
+However, obesity was also identified as a risk factor for hospitalization of patients during the H1N1 influenza epidemic in 2009 [@doi:10.1001/jama.2009.1583; @doi:10.4065/mcp.2010.0166], thus there is a need to explore the mechanistic links between obesity and outcomes of viral respiratory infections [@doi:10.1136/thoraxjnl-2012-203009; @doi:10.1016/j.pupt.2012.04.006].
 Dementia and cancer were also associated with the risk of death in an analysis of a large number (more than 20,000) COVID-19 patients in the United Kingdom [@doi:10.1136/bmj.m1985].
 It should be noted that comorbid conditions are inextricably tied to age, as conditions tend to be accumulated over time, but that the prevalence of individual comorbidities or of population health overall can vary regionally [@doi:10.1073/pnas.2008760117].
 Several comorbidities that are highly prevalent in older adults, such as COPD, hypertension, cardiovascular disease, and diabetes, have been associated with CFRs upwards of 8% compared to an estimate of 1.4% in people without comorbidities [@doi:10.1111/jgs.16472; @url:https://www.who.int/publications/i/item/report-of-the-who-china-joint-mission-on-coronavirus-disease-2019-(covid-19)]. <!-- will this url work for Manubot?-->
 Therefore, both age and health are important considerations when predicting the impact of COVID-19 on a population [@doi:10.1073/pnas.2008760117].
-However, other associations may exist, such as patients with sepsis having higher SOFA scores -- in fact, SOFA was developed for the assessment of organ failure in the context of sepsis, and the acronym originally stood for Sepsis-Related Organ Failure Assessment [@doi:10.1007/BF01709751; @doi:10.1001/jama.2016.0287].
-Additionally, certain conditions are likely to be more prevalent under or exacerbated by social conditions, especially poverty, as is discussed further below.
+However, other associations may exist, such as patients with sepsis having higher SOFA scores -- in fact, SOFA was developed for the assessment of organ failure in the context of sepsis, and the acronym originally stood for sepsis-related organ failure assessment [@doi:10.1007/BF01709751; @doi:10.1001/jama.2016.0287].
+Indeed, vitamin D insufficiency and deficiency has also been linked to COVID-19 incidence and mortality, particularly in Black, Asian, and minority ethnic (BAME) groups [@doi:10.3390/nu13030740; @doi:2102.02250v1; 10.1001/jamanetworkopen.2021.4117; @doi:10.1016/j.mayocp.2021.02.014].
```

Comment:
These two DOIs look incomplete: `@doi:2102.02250v1; 10.1001/jamanetworkopen.2021.4117`

Reply:
```suggestion
Indeed, vitamin D insufficiency and deficiency has also been linked to COVID-19 incidence and mortality, particularly in Black, Asian, and minority ethnic (BAME) groups [@doi:10.3390/nu13030740; @arxiv:2102.02250v1; @doi:10.1001/jamanetworkopen.2021.4117; @doi:10.1016/j.mayocp.2021.02.014].
```

Reply:
Thanks @agitter

Reply:
Thanks for updating.  I had missed that the arXiv paper was the nutraceuticals manuscript.  Is it okay with you to switch it to the alias `@individual-nutraceuticals`?  That will make it easier to change all references to the journal version once it appears.

Reply:
Perfect yes